src/core/core_messages_scheme.json

```diff
@@ -130,13 +130,7 @@
           "userDHKey": {"type": "string"},
           "gasLimit": {"type": "number"},
           "contractAddress": {"type": "string"},
-          "preCode": {
-            "type": "array",
-            "items": {
-              "type": "number",
-              "minItems": 1
-            }
-          }
+          "preCode": {"type": "string"}
```

Comment:
I'm not sure how did that happen but I think the right thing to do is to change it in core to `Vec<u8>`